autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { AppBar, Toolbar, Typography, Button, IconButton } from '@material-ui/core';
+// import GoogleLogin from './GoogleButton';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+}));
+
+const style = {
+    background : '#631717'
+}
+
+const NavBar: React.SFC = function App() {
+  const classes = useStyles(useStyles);
+
+  return (
+    <div className={classes.root}>
+      <AppBar position="static" style={style}>
```

Comment:
```suggestion
      <AppBar position="static" color="primary">
```

Change this to use the global theme, rather than a custom style, so we have a unified color scheme & can modify all of the components at once